### ISSUE TITLE
Jenkins Output confusing

### ISSUE THREAD

**Issue by kevina** [ipfs/go-ipfs#5398]
In this build, https://ci.ipfs.team/blue/organizations/jenkins/IPFS%2Fgo-ipfs/detail/PR-5281/25/pipeline/126, it is very non-obvious that anything is wrong.  It is marked as "unstable", but gives no clear indication that anything failed:

![screenshot](https://user-images.githubusercontent.com/1271492/44499711-af2a4400-a674-11e8-9d5b-cd2ccc52d52a.png)

I was going to report that it did not recognize that sharness test did not pass, but apparently it did. I was expecting a red X on that part of the pipeline, but instead everything is marked with a "!".

It was only after closer examination that I discovered a "1" at the top near tests, so I clicked on it, and low and behold that is what is failing.

Very non-obvious in my option.

Is there a way we can fix this to make it more obvious.  Having the sharness test fail if one of the test fail and a red "X" on that part of the pipeline will be very helpful if it is possible.

**Comment by magik6k** [comment]
This is a bug in jenkins itself - https://issues.jenkins-ci.org/browse/JENKINS-39203

We can't easily make sharness fail early because of a few reasons:
* We need to run sharness fully to get test results passed to jenkins
* Tests are executed in parallel (`make -j12`), console output is not exactly useful

IMO the setup where you see that the CI failed, clicking on jenkins status in PR and then the open the 'tests' tab on top to see cleanly what failed is about as simple/fast as it can get.

TL;DR pipeline display is a display issue, will fix itself once fixed in Jenkins

**Comment by kevina** [comment]
Reopening and labeling as blocked.  Feel free to move the issue to `ipfs/testings` or a more appropriate place then close again, but I want _some_ issue open to track this until the bug is fixed upstream.

**Comment by Stebalien** [comment]
Closing as we've deprecated jenkins.
